UnityPrefabs/Assets/ExplosionParticle.cs

```diff
@@ -0,0 +1,39 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ExplosionParticle : MonoBehaviour
+{
+
+    public float explosionForce = 300f;
+    public float lifetime = 1.5f;
```

Comment:
variables for force behind explosion event and how long particles last after explosion